sites/public/src/components/account/StatusItem.tsx

```diff
@@ -30,7 +30,7 @@ const StatusItem = (props: StatusItemProps) => {
           <h3 className={styles["status-item__title"]}>{props.listingName}</h3>
           <p className={styles["status-item__status"]}>
             {props.strings?.status ?? t("application.status")}:{" "}
-            <Tag variant="primary">
+            <Tag variant="highlight-cool">
```

Comment:
I don't know if it's just my monitor, but this color is really hard to see. Is that the recommended tag?
![image](https://github.com/metrotranscom/doorway/assets/42942267/55c7a376-e0ef-4ab9-ad5c-502b7c51ab42)

Reply:
https://github.com/metrotranscom/doorway/pull/477#issuecomment-1945126549 This is the design discrepancy but it could look like this 
<img width="126" alt="Screenshot 2024-02-14 at 4 40 08 PM" src="https://github.com/metrotranscom/doorway/assets/53269332/c3d7e5b1-7718-4d47-bd87-85d9f3e022ff">

Reply:
Decided to go with the more accessible version I posted above